### PR TITLE
Move alpha provision k3d impl to stable cmd

### PR DESCRIPTION
**Pull request by lindnerby**
**Description**

Since k3d v5 will be our new baseline for `kyma provision k3d`, the alpha command which works with version 5 should be the new stable.

**Related issue(s)**
* https://github.com/kyma-project/cli/issues/1092

### CODE REVIEW

**Review by chrkl — APPROVED** [review]

File: cmd/kyma/provision/k3d/cmd.go [hunk 1]
```diff
@@ -22,27 +22,27 @@ type command struct {
...
-	cmd.Flags().StringSliceVarP(&o.ServerArgs, "server-arg", "s", []string{}, "One or more arguments passed to the Kubernetes API server (e.g. --server-arg='--alsologtostderr')")
-	cmd.Flags().StringSliceVarP(&o.AgentArgs, "agent-arg", "a", []string{}, "One or more arguments passed to the k3d agent command on agent nodes (e.g. --agent-arg='--alsologtostderr')")
+	cmd.Flags().StringSliceVarP(&o.K3sArgs, "k3s-arg", "s", []string{}, "One or more arguments passed from k3d to the k3s command (format: ARG@NODEFILTER[;@NODEFILTER])")
 	cmd.Flags().DurationVar(&o.Timeout, "timeout", 5*time.Minute, `Maximum time for the provisioning. If you want no timeout, enter "0".`)
 	cmd.Flags().StringSliceVarP(&o.K3dArgs, "k3d-arg", "", []string{}, "One or more arguments passed to the k3d provisioning command (e.g. --k3d-arg='--no-rollback')")
 	cmd.Flags().StringVarP(&o.KubernetesVersion, "kube-version", "k", "1.20.11", "Kubernetes version of the cluster")
+	cmd.Flags().StringSliceVar(&o.UseRegistry, "registry-use", []string{}, "Connect to one or more k3d-managed registries. Kyma automatically creates a registry for serverless images.")
```
Comment (majakurcius) [inline cmd/kyma/provision/k3d/cmd.go:44]:
```suggestion
	cmd.Flags().StringSliceVar(&o.UseRegistry, "registry-use", []string{}, "Connect to one or more k3d-managed registries. Kyma automatically creates a registry for Serverless images.")
```
Unless I got it wrong and it's not what we mean here.

File: docs/gen-docs/kyma_provision_k3d.md [hunk 2]
```diff
@@ -15,14 +15,15 @@ kyma provision k3d [flags]
...
-  -s, --server-arg strings    One or more arguments passed to the Kubernetes API server (e.g. --server-arg='--alsologtostderr')
-      --timeout duration      Maximum time for the provisioning. If you want no timeout, enter "0". (default 5m0s)
-      --workers int           Number of worker nodes (k3d agents) (default 1)
+      --k3d-arg strings        One or more arguments passed to the k3d provisioning command (e.g. --k3d-arg='--no-rollback')
+  -s, --k3s-arg strings        One or more arguments passed from k3d to the k3s command (format: ARG@NODEFILTER[;@NODEFILTER])
+  -k, --kube-version string    Kubernetes version of the cluster (default "1.20.11")
+      --name string            Name of the Kyma cluster (default "kyma")
+  -p, --port strings           Map ports 80 and 443 of K3D loadbalancer (e.g. -p 80:80@loadbalancer -p 443:443@loadbalancer) (default [80:80@loadbalancer,443:443@loadbalancer])
+      --registry-port string   Specify the port on which the k3d registry will be exposed (default "5001")
+      --registry-use strings   Connect to one or more k3d-managed registries. Kyma automatically creates a registry for serverless images.
```
Comment (majakurcius) [inline docs/gen-docs/kyma_provision_k3d.md:24]:
```suggestion
      --registry-use strings   Connect to one or more k3d-managed registries. Kyma automatically creates a registry for Serverless images.
```
Unless I got it wrong and it's not what we mean here.

**Review by majakurcius — APPROVED** [review]
I've added 2 comments but I'm going to approve it to help unjam the tests. If possible, apply the comments now, if not, do it in the next PR.

**Comment by lindnerby** [comment]
> I've added 2 comments but I'm going to approve it to help unjam the tests. If possible, apply the comments now, if not, do it in the next PR.

Will do with a follow-up, when removing the alpha completly. Thank you!

**Pull request merged**